page_objects/body_page.py

```diff
@@ -29,8 +36,64 @@ def press_add_to_cart_button(self):
         self.find_element(*self.add_to_cart).click()
         self.wait_element(*self.confirmation_modal)
 
-    
     def get_item_cart(self):
         self.wait_element(*self.cart_amount)
         return self.find_element(*self.cart_amount).text
 
+    def getMenuElements(self): #este método devuelve todos los elementos del menú
+        return self.find_elements(*self.menu_elements)
+
+    def getMenuElementByTitle(self,m_title): #este método devuelve dinámicamente un elemento del menú según el título que se pase por parámetro
+        elements = self.getMenuElements()
+        menuelement = None
+        for elem in elements:
```

Comment:
La idea detrás de estos cambios está interesante, pero se pueden manejar de una forma mas "prolija" utilizando la función `filter` y definiendo otra función que realice la comparación por attribute. Fijate https://www.programiz.com/python-programming/methods/built-in/filter por mas info